### PR TITLE
[ButtonBar, NavigationBar] Add BUILD file for kokoro

### PR DESCRIPTION
**Pull request by romoore**
ButtonBar ColorThemer depends on NavigationBar
NavigationBar depends on ButtonBar

Have to add both at once

### CODE REVIEW

**Review by jverkoey — DISMISSED** [review]

**Review by jgunaratne — DISMISSED** [review]

**Comment by romoore** [comment]
Can't seem to get past MDFTextAccessibility import in Swift unit test.

```
ERROR:/.../repo/components/NavigationBar/BUILD:77:1: Compiling Swift module unit_test_swift_sources (1 files) failed (Exit 1).
components/NavigationBar/tests/unit/NavigationBarAccessibilityMutatorTestColorTest.swift:18:8: error: no such module 'MDFTextAccessibility'
import MDFTextAccessibility
       ^
```

**Comment by romoore** [comment]
Yaaaaaassss!!!!!

**Comment by romoore** [comment]
Ready for re-review. Please take a look.

File: components/NavigationBar/tests/unit/NavigationBarAccessibilityMutatorTestColorTest.swift [hunk 1]
```diff
@@ -15,8 +15,13 @@
...
 
 import XCTest
-import MaterialComponents
+import MaterialComponents.MaterialNavigationBar
+
+#if IS_BAZEL_BUILD
+import material_text_accessibility_ios_MDFTextAccessibility
+#else 
 import MDFTextAccessibility
+#endif // IS_BAZEL_BUILD
```
Comment (jverkoey) [inline components/NavigationBar/tests/unit/NavigationBarAccessibilityMutatorTestColorTest.swift]: We should be able to drop this and the flag with 4cb991b649ae856de73cb217b518bd50f539016a landed.

**Comment by romoore** [comment]
Blocked, waiting for rewrite rules in #2623

**Comment by romoore** [comment]
Now blocked on
* [MDFI18N #36](https://github.com/material-foundation/material-internationalization-ios/pull/36)
* #2640

**Comment by romoore** [comment]
Ready for re-review. Please take a look.

**Comment by romoore** [comment]
Gentle ping to review.

File: components/ButtonBar/BUILD [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+)
+
+mdc_objc_library(
+    name = "private",
+    hdrs = native.glob(["src/private/*.h"]),
+    includes = ["src/private"],
+    visibility = [":test_targets"],
+)
+
+package_group(
```
Comment (jverkoey) [inline components/ButtonBar/BUILD]: package_group!

Comment (romoore) [reply components/ButtonBar/BUILD]: Done.

**Comment by romoore** [comment]
Ready for re-re-review

**Review by jverkoey — APPROVED** [review]

**Comment by romoore** [comment]
The latest change has no impact on Travis. Merging as Kokoro has passed.

**Pull request merged**